src/views/DocumentationTopic.vue

```diff
@@ -23,6 +23,7 @@
             :interface-language="topicProps.interfaceLanguage"
             :technologyUrl="technology.url"
             :api-changes-version="store.state.selectedAPIChangesVersion"
+            :currentLocale="$i18n.locale"
```

Comment:
Is this ever used? It probably is but I think I'm just not seeing it.

Reply:
it belongs to this other PR, ops, I deleted it here